frontend/src/metabase/visualizations/lib/settings/graph.js

```diff
@@ -402,7 +402,6 @@ export const GRAPH_AXIS_SETTINGS = {
     section: t`Axes`,
     title: t`X-axis scale`,
     widget: "select",
-    default: "ordinal",
```

Comment:
Unrelated to the fix, but I noticed it wasn't doing anything since `getDefault` is used.